spec/mongoid/validatable/uniqueness_spec.rb

```diff
@@ -19,34 +19,42 @@
         context "when persisting to another collection" do
 
           before do
-            Dictionary.with(collection: "dicts").create(name: "websters")
+            Dictionary.with(collection: "dicts") do |klass|
+              klass.create(name: "websters")
+            end
           end
 
           context "when the document is not valid" do
 
             let(:websters) do
-              Dictionary.with(collection: "dicts").new(name: "websters")
+              valid = nil
```

Comment:
Why did this test need to change? Shouldn't the block be returning the result of the yield?